packages/core/src/index.ts

```diff
@@ -1,8 +1,9 @@
-export * from '@envelop/types';
-export * from './create';
-export * from './plugins/use-logger';
-export * from './plugins/use-timing';
-export * from './plugins/use-schema';
-export * from './plugins/use-error-handler';
-export * from './plugins/use-extend-context';
-export * from './plugins/use-payload-formatter';
+export * from '@envelop/types';
+export * from './create';
+export * from './plugins/use-logger';
+export * from './plugins/use-timing';
+export * from './plugins/use-schema';
+export * from './plugins/use-error-handler';
+export * from './plugins/use-extend-context';
+export * from './plugins/use-payload-formatter';
+export * from './plugins/use-masked-errors';

```

Comment:
crlf commits?

Reply:
ahhh f windows :(

Reply:
if husky is added with pretty-quick, this issue won't ever happen

Reply:
fixed 🕺